DuckDuckGo/Menus/MainMenu.swift

```diff
@@ -101,9 +101,12 @@ final class MainMenu: NSMenu {
 
     let helpMenu = NSMenu(title: UserText.mainMenuHelp)
     let aboutMenuItem = NSMenuItem(title: UserText.about, action: #selector(AppDelegate.showAbout))
+    let setAsDefaultMenuItem = NSMenuItem(title: UserText.setAsDefaultBrowser + "...", action: #selector(AppDelegate.setAsDefault))
```

Comment:
"…" symbol should be used in menu items instead of 3 dots "..."
(the three dots also got to some other values in UserText 😔)

Reply:
Yeah, let me update it. I used three dots because they were being used in other places 😬